raise.rb:3
raise "abc"
# # ********************************************************
# # This tests that the debugger doesn't step into itself
# # when the application doesn't terminate the right way.
# # ********************************************************
# set debuggertesting on
Currently testing the debugger is on.
# catch x
NameError Exception: undefined local variable or method `x' for main:Object
# catch ZeroDivisionError
Catch exception ZeroDivisionError.
# info catch
ZeroDivisionError
# catch 5
Warning 5 is not known to be a Class
Catch exception 5.
# step
	./example/raise.rb:3:in `(root)'
	org/jruby/debug/RubyDebugger.java:209:in `debug_load'
	./tdebug.rb:1:in `(root)'
	./tdebug.rb:61:in `debug_program'
	org/jruby/RubyKernel.java:1052:in `load'
	./tdebug.rb:252:in `(root)'
	../rdbg.rb:1:in `(root)'
Uncaught exception: abc
